api/v1alpha1/systemconfiguration_types.go

```diff
@@ -59,6 +60,12 @@ type SystemConfigurationSpec struct {
 
 	// StorageNodes is the list of storage nodes on the system
 	StorageNodes []SystemConfigurationStorageNode `json:"storageNodes,omitempty"`
+
+	// Ports is the list of ports available for communication between nodes in the system.
+	// Valid values are single integers, or a range of values of the form "START-END" where
+	// START is an integer value that represents the start of a port range and END is an
+	// integer value that represents the end of the port range (inclusive).
+	Ports []intstr.IntOrString `json:"ports,omitempty"`
```

Comment:
Should we include containers in the name? e.g. ContainerPorts?

Reply:
I'm designing this so anyone who needs a port can request one. It's independent of the container feature